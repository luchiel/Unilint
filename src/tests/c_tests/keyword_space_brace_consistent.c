for
(int i = 0; i < n; ++i) {}

for(;;) {}
for (;;) {}