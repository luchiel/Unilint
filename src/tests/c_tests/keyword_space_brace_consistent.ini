[spaces_and_newlines]
space_between_keyword_and_brace = consistent