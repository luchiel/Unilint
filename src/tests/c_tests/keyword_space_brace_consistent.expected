5:4: space after for
